File: javascript/app/domain/rental.js
function Rental(movie, daysRented) {
	this.getDaysRented = getDaysRented;
	this.getMovie = getMovie;
	
	function getMovie() {
		return movie;
	}
	
	function getDaysRented() {
		return daysRented;
	}
	
}

module.exports = Rental;

File: csharp/Soat.CleanCoders.VideoStore.Model/Rental.cs
﻿namespace Soat.CleanCoders.VideoStore.Model
{
    public class Rental
    {
        public int DaysRented { get; }
        public virtual Movie Movie { get; }

        public Rental(Movie movie, int daysRented)
        {
            this.Movie      = movie;
            this.DaysRented = daysRented;
        }
    }
}
